src/cmake/config/win-vs.txt

```diff
@@ -154,18 +154,6 @@ if (USE_AVX2)
         /arch:AVX2                          # Advanced Vector Extensions 2
     )
 endif ()
-if (USE_FMA)
```

Comment:
It doesn't look like F16C is supported by any CPU yet. In any case, AVX2 support certainly doesn't imply F16C support. However I checked and it *does* look like AVX2 support implies FMA support, so there's at least that.